.gitlab/build_toss4.yml

```diff
@@ -61,11 +61,11 @@ toss4-gcc_10_3_1-src-no-tribol:
     ALLOC_TIME: "30"
   extends: .src_build_on_toss4
 
-toss4-gcc_10_3_1-src-no-sundials:
+toss4-gcc_10_3_1-src-no-optional-solvers:
   variables:
     COMPILER: "gcc@10.3.1"
     HOST_CONFIG: "ruby-toss_4_x86_64_ib-${COMPILER}.cmake"
-    EXTRA_CMAKE_OPTIONS: "-DENABLE_BENCHMARKS=ON -USUNDIALS_DIR"
+    EXTRA_CMAKE_OPTIONS: "-DENABLE_BENCHMARKS=ON -USUNDIALS_DIR -UPETSC_DIR"
```

Comment:
Not really connected to this PR but do we want benchmarks enabled for these jobs?

Reply:
i think it makes sense to at least build them here, so make sure they compile. at the moment they do not run in ci. at the moment they aren't too lengthy, but assuming the benchmarks take longer, it would make sense to run them on merge or weekly, rather than on every push.

Reply:
Thats a good point.